### PR TITLE
docs: update mnist with PoT + rounding

### PR DESCRIPTION
**Pull request by jfrery**
closes https://github.com/zama-ai/concrete-ml-internal/issues/3970

### CODE REVIEW

File: docs/advanced_examples/FullyConnectedNeuralNetworkOnMNIST.ipynb [hunk 1]
```diff
@@ -355,14 +359,14 @@
...
-   "execution_count": 13,
+   "execution_count": 14,
    "metadata": {},
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "Execution time in FHE: 241.30 seconds per sample\n",
+      "Execution time in FHE: 4.05 seconds per sample\n",
```
Comment (RomanBredehoft) [inline docs/advanced_examples/FullyConnectedNeuralNetworkOnMNIST.ipynb:369]: wow

File: docs/advanced_examples/FullyConnectedNeuralNetworkOnMNIST.ipynb [hunk 2]
```diff
@@ -182,18 +171,17 @@
...
-   "execution_count": 7,
+   "execution_count": 8,
    "metadata": {},
    "outputs": [],
    "source": [
     "params = {\n",
     "    \"module__n_layers\": 2,\n",
     "    \"module__n_w_bits\": 4,\n",
     "    \"module__n_a_bits\": 4,\n",
-    "    \"module__n_accum_bits\": 15,\n",
```
Comment (RomanBredehoft) [inline docs/advanced_examples/FullyConnectedNeuralNetworkOnMNIST.ipynb:193]: no more `module__n_accum_bits` ??

File: docs/advanced_examples/FullyConnectedNeuralNetworkOnMNIST.ipynb [hunk 3]
```diff
@@ -332,15 +336,15 @@
...
+   "execution_count": 13,
    "metadata": {},
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
       "FHE circuit of 12-bits\n",
-      "Key generation time: 5.75 seconds\n"
+      "Key generation time: 18.90 seconds\n"
```
[the remark this answers is not in the document]
Comment (jfrery) [reply docs/advanced_examples/FullyConnectedNeuralNetworkOnMNIST.ipynb:347]: Yes we have more keys to generate.

File: docs/advanced_examples/FullyConnectedNeuralNetworkOnMNIST.ipynb (hunk 2 above)
Comment (jfrery) [reply docs/advanced_examples/FullyConnectedNeuralNetworkOnMNIST.ipynb:193]: PoT + Rounding feature handles the accumulator by itself. Nothing to do yes.

Comment (RomanBredehoft) [reply docs/advanced_examples/FullyConnectedNeuralNetworkOnMNIST.ipynb:193]: I see thanks !

File: docs/advanced_examples/FullyConnectedNeuralNetworkOnMNIST.ipynb (hunk 1 above)
Comment (bcm-at-zama) [reply docs/advanced_examples/FullyConnectedNeuralNetworkOnMNIST.ipynb:369]: indeed

**Review by andrei-stoian-zama — APPROVED** [review]

**Review by fd0r — APPROVED** [review]

**Review by RomanBredehoft — APPROVED** [review]

**Pull request merged**